src/js/rx/components/Disclaimer.jsx

```diff
@@ -1,26 +1,44 @@
 import React from 'react';
+import classNames from 'classnames';
+
 
 class Disclaimer extends React.Component {
   render() {
+    const openClass = classNames(
+      { 'rx-disclaimer--open': this.props.isOpen }
+    );
+
+    const openButton = classNames(
+      { 'fa-chevron-down': !this.props.isOpen },
+      { 'fa-chevron-up': this.props.isOpen }
```

Comment:
Can combine class names here too:

    const buttonIconClass = classNames({
      'fa': true,
      'fa-chevron-down': !this.props.isOpen,
      'fa-chevron-up': this.props.isOpen
    });